pkg/convert/testdata/programs/heredoc_template_expr/main.tf

```diff
@@ -0,0 +1,8 @@
+locals {
+  a_list = ["a", "b", "c"]
+  join_template_expr = <<EOT
+%{for v in local.a_list~}
+${v}
+%{endfor~}
+EOT
+}
```

Comment:
Worth adding some more test cases with different heredoc delimiters, heredocs that are "just" simple strings (no control blocks), etc.?

Reply:
This is included in the "expressions" already